lib/Resque.php

```diff
@@ -262,118 +199,4 @@ public static function queues()
-	 * If the Job matches, counts otherwise puts it in a requeue_queue
-	 * which at the end eventually be copied back into the original queue
-	 *
-	 * @private
-	 *
-	 * @param string $queue The name of the queue
-	 * @param array $items
-	 * @return integer number of deleted items
-	 */
-	private static function removeItems($queue, $items = Array())
-	{
-		$counter = 0;
-		$originalQueue = 'queue:'. $queue;
-		$tempQueue = $originalQueue. ':temp:'. time();
-		$requeueQueue = $tempQueue. ':requeue';
-
-		// move each item from original queue to temp queue and process it
-		$finished = false;
-		while (!$finished) {
-			$string = self::redis()->rpoplpush($originalQueue, self::redis()->getPrefix() . $tempQueue);
-
-			if (!empty($string)) {
-				if(self::matchItem($string, $items)) {
-					self::redis()->rpop($tempQueue);
-					$counter++;
-				} else {
-					self::redis()->rpoplpush($tempQueue, self::redis()->getPrefix() . $requeueQueue);
-				}
-			} else {
-				$finished = true;
-			}
-		}
-
-		// move back from temp queue to original queue
-		$finished = false;
-		while (!$finished) {
-			$string = self::redis()->rpoplpush($requeueQueue, self::redis()->getPrefix() .$originalQueue);
-			if (empty($string)) {
-			    $finished = true;
-			}
-		}
-
-		// remove temp queue and requeue queue
-		self::redis()->del($requeueQueue);
-		self::redis()->del($tempQueue);
-
-		return $counter;
-	}
-
-	/**
-	 * matching item
-	 * item can be ['class'] or ['class' => 'id'] or ['class' => {:foo => 1, :bar => 2}]
-	 * @private
-	 *
-	 * @params string $string redis result in json
-	 * @params $items
-	 *
-	 * @return (bool)
-	 */
-	private static function matchItem($string, $items)
-	{
-	    $decoded = json_decode($string, true);
-
-	    foreach($items as $key => $val) {
-		# class name only  ex: item[0] = ['class']
-		if (is_numeric($key)) {
-		    if($decoded['class'] == $val) {
-			return true;
-		    }
-		# class name with args , example: item[0] = ['class' => {'foo' => 1, 'bar' => 2}]
-    		} elseif (is_array($val)) {
-		    $decodedArgs = (array)$decoded['args'][0];
-		    if ($decoded['class'] == $key &&
-			count($decodedArgs) > 0 && count(array_diff($decodedArgs, $val)) == 0) {
-			return true;
-			}
-		# class name with ID, example: item[0] = ['class' => 'id']
-		} else {
-		    if ($decoded['class'] == $key && $decoded['id'] == $val) {
-			return true;
-		    }
-		}
-	    }
-	    return false;
-	}
-
-	/**
-	 * Remove List
-	 *
-	 * @private
-	 *
-	 * @params string $queue the name of the queue
-	 * @return integer number of deleted items belongs to this list
-	 */
-	private static function removeList($queue)
-	{
-	    $counter = self::size($queue);
-	    $result = self::redis()->del('queue:' . $queue);
-	    return ($result == 1) ? $counter : 0;
-	}
-
-	/*
-	 * Generate an identifier to attach to a job for status tracking.
-	 *
-	 * @return string
-	 */
-	public static function generateJobId()
-	{
-		return md5(uniqid('', true));
-	}
```

Comment:
Not strictly vital, but consistency is nice, so putting this in a method rather than duplicating it everywhere is helpful.